style.css

```diff
@@ -0,0 +1,476 @@
+}
+
+.content__one-big {
+  font-size: 35px;
+  font-family: 'montserratsemibold', sans-serif;
+  margin: 0 5px 0 0;
+}
+
+.content__one-small {
+  font-size: 20px;
+  font-family: 'montserratsemibold', sans-serif;
+}
+
+.content__item-two {
+  font-size: 12px;
+  font-family: 'montserratregular', sans-serif;
+}
+
+.button__out {
+  border: none;
+  display: flex;
+  align-items: center;
+}
+
+button:hover {
+  opacity: 0.5;
+}
+
+.button__pic {
+  margin-right: 5px;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.button__redwheel {
+  display: none;
+}
+
+.button__border {
+  display: flex;
+  align-items: center;
+  border: 2px solid #e51937;
+  color: #e51937;
+}
+
+.button__text {
+  text-transform: uppercase;
+}
+
+.main__pointer {
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+section {
+	background-image: url("pics/teslamodels.png");
+  background-attachment: inherit;
+  background-repeat: no-repeat;
+}
+
+.sec__container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.section__pic {
+  display: none;
+ }
+
+.section__title {
+  color: #e51937;
+  font-size: 70px;
+  font-family: 'montserratsemibold', sans-serif;
+}
+
+.article {
+  left: 69%;
+  width: 31%;
+  color: white;
+  background-color: #e51937;
+}
+
+.article__info {
+   padding: 40px 30px;
+}
+
+.article__charset {
+  margin: 30px 0;
+  display: flex;
+  flex-direction: column;
+  padding: 0 25px;
+}
+
+.article__top {
+  padding: 10px 35px;
+}
+
+.article__title {
+  font-size: 30px;
+}
+
+.charset {
+  display: flex;
+  padding: 20px 0;
+  align-items: center;	
+}
+
+.charset__item {
+  margin: 0 10px;	
+}
+
+.charset__item-one {
+  display: flex;
+  align-items: baseline;
+  font: 20px 'montserratsemibold',sans-serif;
+}
+
+.charset__item-two {
+  display: flex;
+  padding: 0 10px 0 0;
+}
+
+.charset__one-big {
+  font-size: 35px;
+  font-family:  'montserratsemibold', sans-serif;	
+}
+
+.charset__one-small {
+  margin: 0 10px;
+}
+
+.charset__button-out {
+  border: none;
+  color: white;
+  display: flex;
+}
+
+.charset__button-pic {
+  margin-right: 5px;
+}
+
+.charset__button-border {
+  border: 2px solid white;
+  color: white;
+}
+
+.buttons {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 20px 0;
+	
+}
+
+.button__text {
+  text-transform: uppercase;
+}
+
+footer {
+  display: flex;
+  color: white;
+}
+
+.footer__box {
+  background-color: #E51937;
+  width: 31%;
+}
+
+.footer__inform {
+  padding: 20px;
+}
+ .footer__picture {
+  position: relative;
+ }
+
+.footer__car {
+  position: relative;
+  z-index: 2;
+  top: 25%;
+  left: -10%;
+  }
+
+.footer__model {
+  position: absolute;
+  z-index: 1;
+  top: 20%;
+  left: 4%;
+ }
+
+.footer__footer {
+  margin: 30px;
+}
+ 
+.footer__title {
+  font: 30px 'montserratmedium';
+  padding: 10px;
+ }
+
+.footer__text {
+  padding: 0 0 30px 0;
+  margin: 10px;
+}
+ 
+.footer__info {
+   margin: 30px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)